translations/en-us.yml

```diff
@@ -514,13 +514,14 @@ validationErrors:
     affirm_terms: 'You must read and agree to the Terms of Use and Privacy Policy.'
     min_subjects: 'You must select at least one subject.'
     node_license_invalid: 'Invalid required fields for the license'
-    node_license_missing_fields: 'The following required fields are missing: {missingFields}'
+    node_license_missing_fields: 'The following required field(s) are missing: {missingFields}'
```

Comment:
Please use `ember-intl` ICU syntax for this. A là https://github.com/CenterForOpenScience/ember-osf-web/pull/979/files#diff-d3d181a8064c420f27753c53d181a5c6R522. You may need to add another key for this translation because it is used in multiple places. That way we have `The following required field is missing` or `The following required fields are missing`. `The following required field(s) are missing` does not look right.